svc/FetchService.js

```diff
@@ -40,6 +56,9 @@ export class FetchService {
      * @param {string} [opts.contentType] - value to use in the Content-Type header in the request.
      *     If not specified, contentType is set based on method: 'application/x-www-form-urlencoded'
      *     for POSTs, 'text/plain' otherwise.
+     * @param {Object|Headers} [opts.headers] - headers to send with this request. If an Object,
+     *      will be merged with existing default headers. Alternatively, provide an
+     *      instantiated Headers object to replace entirely.
```

Comment:
Tempted to remove contentType as a top-level input to this API - seems like the dev could/should just pass it as a header since we now support the easy-to-use object form and that is where it will land anyway.  Could note in the headers doc comment that "a Content-Type header will be set if not provided by the caller directly or via one of the  dedicated `xxxJson` methods on this service."

This would be another breaking change but I think it's worth it.  If we have reason to believe this is used in more than one or two places, we could add a warnIf to easy the API transition. (I am in favor of those when they allow us to take cleanups we want to make while not being too annoying during upgrades.)